### PR TITLE
Updated Demo & OnPlayerJoin

### PR DESCRIPTION
**Pull request by momintlh**
Added Jumping to the demo game
Updated OnPlayerJoin.

### CODE REVIEW

File: Assets/PlayroomKit/PlayroomKit.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public class InitOptions
         private static Action InsertCoinCallback = null;
 
         [DllImport("__Internal")]
-        private static extern void InsertCoinInternal(Action callback, string options, Action<string> onQuitInternalCallback);
+        private static extern void InsertCoinInternal(Action callback, string options, Action<string> onJoinCallBack ,Action<string> onQuitInternalCallback);
```
Comment (SaadBazaz) [inline Assets/PlayroomKit/PlayroomKit.cs:40]: Small nits:
1. Rename `onJoinCallback` to `onPlayerJoinInternalCallback`
2. Fix space between comma and `onJoinCallback`

File: Assets/PlayroomKit/PlayroomKit.cs [hunk 2]
```diff
@@ -75,36 +75,56 @@ private static string SerializeInitOptions(InitOptions options)
...
+                Debug.LogError("PlayroomKit is not loaded!. Please make sure to call InsertCoin first.");
             }
             else
             {
-                if (!isPlayRoomInitialized)
+                if (IsRunningInBrowser())
                 {
-                    Debug.LogError("[Mock Mode] Playroom not initialized yet! Please call InsertCoin.");
+                    // onPlayerJoinCallback = playerCallback;
+                    OnJoinCallbacks.Add(playerCallback);
```
Comment (SaadBazaz) [inline Assets/PlayroomKit/PlayroomKit.cs:113]: Rename playerCallback to onPlayerJoinCallback

File: Assets/PlayroomKit/PlayroomKit.cs [hunk 3]
```diff
@@ -75,36 +75,56 @@ private static string SerializeInitOptions(InitOptions options)
             return JsonUtility.ToJson(options);
         }
 
-        [DllImport("__Internal")]
-        private static extern void OnPlayerJoinInternal(Action<string> callback);
+        // [DllImport("__Internal")]
+        // private static extern void OnPlayerJoinInternal(Action<string> callback);
+
+        // private static Action<Player> onPlayerJoinCallback = null;
+        private static List<Action<Player>> OnJoinCallbacks = new();
```
Comment (SaadBazaz) [inline Assets/PlayroomKit/PlayroomKit.cs:82]: Rename onJoinCallbacks to onPlayerJoinCallbacks

**Review by SaadBazaz — CHANGES_REQUESTED** [review]
Great job! Just some small changes. It's important to use the correct language, so that it's easy for future maintainers.

**Comment by SaadBazaz** [comment]
Add this demo game code to the docs: https://github.com/asadm/playroom-docs/blob/main/pages/usage/unity.mdx

**Comment by momintlh** [comment]
Updated @SaadBazaz

**Pull request merged**